### ISSUE TITLE
NPM Package

### ISSUE THREAD

**Issue by mhrynenko** [witnet/elliptic-curve-solidity#37]
Hi!

Nice library to use, I really enjoy it, but could you update [npm package](https://www.npmjs.com/package/elliptic-curve-solidity) with contracts, because there is still contract with `>=0.5.3 <0.7.0` solidity version. It will be nice to use the most up to date lib

Thanks for your time!

**Comment by raulk** [comment]
We're using a GitHub dependency declaration in `package.json` to import the Solidity v0.8.0 compatible version of this library, given that no package has been published to a registry yet.

```json
"elliptic-curve-solidity": "github:witnet/elliptic-curve-solidity#3475478",
```

**Comment by heueristik** [comment]
Yes, please create a GH release for the commit and ideally publish it on the [soldeer package manager](https://soldeer.xyz/)
FYI, we audited `ecAdd` (see https://github.com/anoma/pa-evm/blob/main/audits/2025-11-24_Informal_Systems_RISC_Zero_RM_%26_EVM_Protocol_Adapter.pdf)
